app/pods/publications/publication/route.js

```diff
@@ -58,13 +56,17 @@ export default class PublicationRoute extends Route.extend(AuthenticatedRouteMix
     const pieces = await _case.get('pieces');
     const documentCount = pieces.length;
 
-    const regulationTypes = await this.store.query('regulation-type', {
+    const regulationTypes = this.store.query('regulation-type', {
```

Comment:
Hier zou ik verwachten dat je de `await` nog steeds nodig hebt?

Reply:
Volgens mij is de functie van `hash` al te awaiten, dus dat lijkt me dubbel werk.

Reply:
Dat klopt. Aangezien er een story op de backlog staat om deze model-hook op termijn te refactoren zodat deze een cleaner model (en normaal geen `hash`) bevat, maakt het me momenteel niet uit welke optie we kiezen.